streams.yml

```diff
@@ -59,14 +59,14 @@ etcd_golang:
 
 # This image must only be used by -alt images, as this image cannot be built for aarch64.
 rhel-7-golang:
-  image: openshift/golang-builder:1.16
+  image: openshift/golang-builder:rhel_7_golang_1.17
```

Comment:
Per https://github.com/openshift/ocp-build-data/pull/1147#discussion_r745811931, we're wondering if the golang-builder pullspecs could/should be made absolute, to avoid questions from everyone as to where to find them.

Reply:
sure, why not :)
```suggestion
  image: registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_7_golang_1.17
```

Reply:
do you want to make rhel_8_golang_1.17 absolute too, or will that trigger a mass rebuild that we really don't need right now?